hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/builtin/CustomTypeFactory.java

```diff
@@ -222,7 +222,7 @@ public static SqlColumnMetadata createSqlColumnMetadata(String name, int type, b
     }
 
     public static FieldDescriptor createFieldDescriptor(@Nonnull String fieldName, int type) {
-        FieldType fieldType = FieldType.get((byte) type);
+        FieldKind fieldType = FieldKind.values()[type];
```

Comment:
Minor: maybe change the variable name as well

Reply:
Also, similar to comment below, we should cache the result of `values` and reuse it.